content/blog/2025-10-15/2025-10-15.md

```diff
@@ -0,0 +1,219 @@
+
+The trains and bike paths were very impressive to see. But, at its core, it's about attention and care for people in public places. The streets have benches where people can sit. Car traffic is separated from pedestrian traffic. People see other people riding the trains, bikes, and, of course, walking around. All these designs facilitate a sense of being together in the society. In the US, we were usually stuck in our car, looking at each other as big metal objects, not as humans.
+
+I fell in love with the country pretty quickly.
+
+### Falling in love professionally
+
+The public spaces in the Netherlands are great, but to be honest, I did not have much time to fully experience them because I spent time mostly at the conference venue.
+
+But, I did not regret it because I had a great time at the conference. It was the experience that I was looking for--connecting with people who are passionate about improving how we do research.
+
+In addition, I also liked the European academic system that treats PhD graduate students as workers, not as just students. During the lunch, I had an opportunity to speak with other graduate students. I shared that PhD graduate students in the US do not have 12-month pay and paid leaves. Many European colleagues were surprised to hear that.
+
+The confusion and horror in their faces
+
+All in all, it's clear to me that I want to be in this environment both personally and professionally, and I want to live in it and support it.
+
+That's why I decided to move to the Netherlands. It was still 2019. Long before defending my dissertation.
+
+## Finishing dissertation, job search, dark times, and reclaiming my purpose
+
+I came back from the Netherlands, and I started to work towards my dissertation. At that time, I was aiming to graduate in 2020.
+
+However, the COVID pandemic happened in 2020, and I had to delay my graduation to 2021.
+
+A year later, I graduated \[lisa and I photo\]
+
+But in my year, I was not ready to move to the Netherlands yet. I just graduated. The world was still in a pandemic mode. And I did not have much funding to move.
+
+So I worked at a non-profit organization called Tech Impact for a year as a Data Science Fellow.
+
+Toward the end of my fellowship, I was applying for jobs in the Netherlands. But I did not get any responses. I tried various kinds of jobs, from teaching positions at universities to data analyst positions at industry. But they did not go well.
+
+Even to Booking.com. Looking back, it may not have been a wise choice.
+
+### Moving nevertheless, amid the housing crisis
+
+I applied to various types of jobs in the Netherlands, but I could not get an offer. Nevertheless, I decided to move nevertheless. After all, it was our intention to move, rain, or shine.
```

Comment:
our was suddenly mentioned made me wonder who you were with because everything was I (hehe)